src/vcsparser.core/MeasureConverterListBuilder.cs

```diff
@@ -94,6 +102,8 @@ private void CreateMeasureConvertersForPeriod(List<IMeasureConverter> result, Da
             result.Add(new MeasureConverter<int>(startDate, endDate, metricNumAuthors, new NumberOfAuthorsMeasureAggregator(), a.FilePrefixToRemove));
             result.Add(new MeasureConverter<int>(startDate, endDate, metricNumAuthors10Perc, new NumberOfAuthorsWithMoreThanTenPercentChangesMeasureAggregator(), a.FilePrefixToRemove));
             result.Add(new MeasureConverter<double>(startDate, endDate, metricLineFixedOverChanges, new LinesFixedOverChangedMetricsMeasureAggregator(), a.FilePrefixToRemove));
+            result.Add(new MeasureConverter<int>(startDate, endDate, metricChangesBugDatabse, new NumberOfChangesBugDatabaseMeasureAggregator(), a.FilePrefixToRemove));
+            result.Add(new MeasureConverter<int>(startDate, endDate, metricChangesWithFixesBugDatabse, new LinesChangedBugDatabaseMeasureAggregator(), a.FilePrefixToRemove));

```

Comment:
Typo metricChangesWithFixesBugDatabse

Reply:
Fixed typo in 436638d125bb40b11223df403d8dd9b234d55800